helm/basemap/templates/cronjobs.yaml

```diff
@@ -98,6 +98,10 @@ spec:
               requests:
                 memory: {{ .Values.resources.db.requests.memory | quote }}
                 cpu: {{ .Values.resources.db.requests.cpu | quote }}
+            lifecycle:
+              preStop:
+                exec:
+                  command: ["sh", "-c", "rm -rf /var/lib/postgresql/data/*"]
```

Comment:
This looks barbaric and wrong. How about the suggestion that this should not be a persistent volume claim but rather ephemeral storage instead?